exec/java-exec/src/main/resources/rest/query/query.ftl

```diff
@@ -83,6 +83,16 @@
       Submit
     </button>
     <input type="checkbox" name="forceLimit" value="limit" <#if model.isAutoLimitEnabled()>checked</#if>> Limit results to <input type="text" id="autoLimit" name="autoLimit" min="0" value="${model.getDefaultRowsAutoLimited()?c}" size="6" pattern="[0-9]*"> rows <span class="glyphicon glyphicon-info-sign" title="Limits the number of records retrieved in the query. Ignored if query has a limit already" style="cursor:pointer"></span>
+    <label> Store Format
+      <select name="store.format">
+        <option value="parquet">Parquet</option>
+        <option value="json">JSON</option>
+        <option value="csv">Comma-Separated Values</option>
+        <option value="psv">Pipe-Separated Values</option>
+        <option value="tsv">Tab-Separated Values</option>
```

Comment:
Should these be hard-coded? Other format plugins can offer the ability to write to that format. Can they be used for a `CREATE TABLE`?

Also, it is annoying, but each plugin declares the formats that it supports. As a result, the user could select `csv`, but that format might not be available for a given file storage plugin. Or, there might be two: `csv` and `csvh`.

Long story short, should we get this list from the plugin which can get it from its list of supported writable formats?

Reply:
That could be an interesting bit of future work for someone - I'm not up for it personally.

Reply:
Store formats names are subject of change. We should not hard-core their names. Better off remove this section of code.

Plus still waiting for screenshots.